daemon/chaintopology.c

```diff
@@ -237,7 +238,7 @@ static void try_broadcast(struct lightningd_state *dstate,
 	this_tx = txs[num_txs-1];
```

Comment:
Actually, the first time it didn't remove the tx, so this was sending the first tx twice.  Erk, testing fix now.